.github/workflows/deploy.yml

```diff
@@ -0,0 +1,67 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows:
+      - Unit Test
+      - UI Test
+    branches:
+      - master
+      - development
```

Comment:
I think it should be `main` and `develop` as you set them at the moment.

```suggestion
    branches:
      - main
      - develop
```

Or they can be detected automatically with those keywords 🤔

Reply:
Fixed in 7e224f0